SpatialGDK/Documentation/features.md

```diff
@@ -48,6 +48,10 @@ Actor property handover is a new feature we’re introducing with the SpatialOS
 
 `Singleton Actors` is a new term that describes Actors which contain global state and logic.  Only one of these Actors exists in a game world. The SpatialOS Unreal GDK supports both client accessible and server only singletons (respectively known as public and private singletons). For more information see the documentation on [Singleton Actors]({{urlRoot}}/content/singleton-actors).
```

Comment:
SpatialOS Unreal GDK -> SpatialOS GDK for Unreal Engine

Reply:
Suggest "The GDK" as we know we are in the Unreal GDK.  (I think we have been all round the houses on this naming and come back to where we started! :-) )